mw-webapp/src/pages/authPage/AuthPage.tsx

```diff
@@ -16,7 +16,7 @@ export const AuthPage = () => {
   const [user, setUser] = useState<User | null>(null);
 
   useEffect(() => {
-    handleUserAuthState(setUser);
+    userAuthState(setUser);
     writeNewUserCredentials();
```

Comment:
we should call this function only if user dont exist

Reply:
Done